src/app/shared/launch.service.ts

```diff
@@ -77,6 +77,36 @@ export abstract class LaunchService {
     return `nextflow run https://${workflowPath} -r ${versionName}`;
   }
 
+  /**
+   * This creates the planemo local init command
+   * @param path The GA4GH Tool's path
+   * @param versionName The ToolVersion's name
+   */
+  getPlanemoLocalInitString(workflowPath: string, versionName: string, primaryDescriptorPath: string, testParameterPath: string) {
+    return (
+      `wget -O foo.zip ${Dockstore.API_URI}${ga4ghPath}/tools/` +
```

Comment:
Nit, I'd give it a name like `temp.zip` to better reflect it's purpose.

Reply:
Ah, forgot about that